lib/graphql/metrics/tracer.rb

```diff
@@ -93,10 +93,7 @@ def capture_multiplex_start_time
       end
 
       def capture_lexing_time
-        # GraphQL::Query#result fires `lex` before the `execute_multiplex` event, so sometimes
-        # `pre_context.multiplex_start_time_monotonic` isn't set.
-        lexing_offset_time = pre_context.multiplex_start_time_monotonic || GraphQL::Metrics.current_time_monotonic
```

Comment:
@chrisbutcher this didn't appear to be used at all? The offset is only needed if `time_since_offset` is used, but it's not in either of these two places.